fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/identifiers/NodeResourceConverter.java

```diff
@@ -63,4 +63,9 @@ protected FedoraResource doForward(final Node node) {
     protected Node doBackward(final FedoraResource resource) {
         return resource.getNode();
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
Not supporting `hashCode()` seems like an a sign of a more fundamental issue.
But any ways, the related Sonar issue seems to be gone: http://sonar.fcrepo.org/drilldown/issues/1?&rule=findbugs%3AEQ_COMPARETO_USE_OBJECT_EQUALS&rule_sev=MAJOR&severity=MAJOR